docs/keycodes_us_ansi_shifted.md

```diff
@@ -6,6 +6,10 @@ These keycodes correspond to characters that are "shifted" on a standard US ANSI
 
 Unfortunately, these keycodes cannot be used in Mod-Taps or Layer-Taps, since any modifiers specified in the keycode are ignored.
 
+Additionally, you may run into issues when using Remove Desktop Connection on Windows. Because these codes send shift very fast, Remote Desktop may miss the codes.
```

Comment:
Same as above

Reply:
```suggestion
Additionally, you may run into issues when using Remote Desktop Connection on Windows. Because these codes send shift very fast, Remote Desktop may miss the codes.
```